osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -95,6 +94,9 @@ public class BeatmapInfo : IEquatable<BeatmapInfo>, IJsonSerializable, IHasPrima
         public bool WidescreenStoryboard { get; set; }
         public bool EpilepsyWarning { get; set; }
 
+        public CountdownType Countdown { get; set; } = CountdownType.Normal;
```

Comment:
Hmm, not sure this actually needs to be at a `BeatmapInfo` level. It could potentially just exist in `Beatmap`. The question is how we want to handle this kind of beatmap-level settings going forward I guess, and whether they should all exist in the database or only at parse-time.

Same can be said for many other variables in this class, so it might not need immediate action – just mentioning it while we're here.

Reply:
It doesn't necessarily need to be on `BeatmapInfo`, true. I was mostly following precedent at this point (the fact that there was a countdown setting there already, and even ignoring that, these two would be the only non-databased settings on the setup screen  side from combo colours).

I won't claim to have any insight on which way will be best eventually. Technically an implementation of three countdown (which doesn't exist yet) shouldn't need this databased, decoding would do just fine.

Reply:
I think eventually we'd see only properties that are relevant outside of gameplay in `BeatmapInfo`, but as you say, it's probably fine to leave this here for now and move them all in bulk.